pkgs/build-support/setup-hooks/wrap-gapps-hook.sh

```diff
@@ -35,6 +35,10 @@ wrapGAppsHook() {
     gappsWrapperArgs+=(--prefix $v : "$dummy")
   done
 
+  if [ -d "$prefix/lib/girepository-1.0" ]; then
+    gappsWrapperArgs+=(--prefix GI_TYPELIB_PATH : "$prefix/lib/girepository-1.0")
```

Comment:
This is supposed to be populated via `gobjectIntrospection` setup hook and added in the loop above.

Reply:
When I drop this patch and build IBus, I get:

```
$ result/bin/ibus-setup
Traceback (most recent call last):
  File "/nix/store/rlz1ssvicswx87v2ry7pfff88chw4kic-ibus-1.5.17/share/ibus/setup/main.py", line 38, in <module>
    gi_require_version('IBus', '1.0')
  File "/nix/store/nmr66mmvhhz9qyshzv9ap72g4rjfbw5j-python3-3.6.3-env/lib/python3.6/site-packages/gi/__init__.py", line 130, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace IBus not available
```

I'll rebase it on top of current master and check if it works now, but at least it didn't back when I started this PR.

Reply:
@jtojnar It doesn't work without this patch, even when rebased on top of `master`. Should I open a separate issue? Or is that an adequate fix?

Reply:
I've made the proper fix, see dd0db4e016ccfce8eb6b2ca3f04be71f0babc5ab.